client/src/components/Address/Address.js

```diff
@@ -45,31 +40,39 @@ const Address = () => {
       })
       .catch((err) => {
         setMessage("error in razor pay");
-        nullMessage();
+        setTimeout(() => {
+          setMessage(null);
+        }, 6000);
```

Comment:
These lines of code are repeated multiple times *duplication*
what can we do it avoid this duplication?